### ISSUE TITLE
Remove 300ms click delay on iOS

### ISSUE THREAD

**Issue by bjornhij** [pimatic/pimatic#706]
On iOS the buttons sometime lag, or do not work at all. This probably is due to the 300ms click delay between a button tap and the firing of the event, that is used to detect if the user wants to double click.

The fastclick library eliminates this delay (https://github.com/ftlabs/fastclick). I 'hacked' the library into my pimatic and for me the delay is now gone.

Is anyone else experiencing lag on the button taps? And is using the fastclick library the correct way to solve this? If so, I can create a pull request, but I'll probably need some guidance on how to implement (without a hack).

**Comment by sweetpi** [comment]
Please add a pull request. I then will take a deeper look into it.

**Comment by bjornhij** [comment]
Three things need to be done to the pimatic-mobile-frontend code:

1) We need to add fastclick.js to the app/js folder;
2) We must load the fastclick.js script (in mobile-frontend.coffee, near line 167 - in the same list where among others jquery is loaded);
3) We have to initialize fastclick with a few lines of javascript. I'm not sure where to do this, so I added the lines to fastclick.js (at the very end).

I've just created a pull request with these changes, let me know what I can do to implement this better.
